vbf/templates/vbf_template_pretty_withphotons.txt

```diff
@@ -38,7 +38,19 @@ bin                                      vbf_@YEAR_signal  vbf_@YEAR_signal  vbf
 process                                  -1                0                 1                 2                 3                 4                13               5                 6                 7                 8                   2                 1                 9                 10                2                 1                 9                 10                1                 13                 2                       4                       5                       6                       8                       1                    2                       4                       5                       6                       8                       11                12                13
 rate                                     -1                -1                -1                -1                1                 -1               -1               1                 -1                1                 1                  -1                -1                 1                 1                 -1                -1                1                 1                 -1                 -1                -1                      -1                       1                      -1                       1                      -1                   -1                      -1                       1                      -1                       1                       1                 1                 -1
-----------------------------------------------------------------------------------------------------------------------------------                -                                                                                                                                                                                                                                                            -
+----------------------------------------------------------------------------------------------------------------------------------                  -                                                                                                                                                                                                                                                             -
+jer                       shape          1                 1                 1                 1                 -                 -                -                -                 -                 -                 -                   1                 1                 -                 -                 1                 1                 -                 -                 1                 1                  1                       1                       -                       1                       -                       1                    1                       1                       -                       1                       -                       -                 -                 -
```

Comment:
I belive ```qcd_zll``` in the SR should also be affected by JES/JER? This is separate from ```qcd_zjets```, which is estimated via TFs. Please also check again other small processes in the CRs.

Reply:
Ah sorry I think I missed it. I'll turn on qcd_zll and ewk_zll in SR as well, and check the other processes and regions, thanks for warning!